examples/PythiaConnector.cpp

```diff
@@ -46,72 +46,48 @@ PythiaConnector::PythiaConnector(const char* fname) : m_store(podio::EventStore(
   m_store.setReader(&m_reader);
 }
 
-papas::ListParticles PythiaConnector::makePapasParticlesFromGeneratedParticles(const fcc::MCParticleCollection* ptcs) {
+void PythiaConnector::makePapasParticlesFromGeneratedParticles(const fcc::MCParticleCollection* ptcs,
```

Comment:
Hi Colin
I completely agree that it could be simpler. However, if I am to match the pdebug outputs from python, I must first sort the generated particles and then create the papas particles from the sorted gen particles. Otherwise the identifiers of the particles differ between papas and cpp. 
Would it be better to change the way it works in Python (and implement your comments above)?